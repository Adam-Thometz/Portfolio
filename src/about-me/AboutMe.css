.AboutMe {
  width: 75%;
  margin: 0 auto;
  background-color: rgba(28, 64, 141, 0.479);
  box-shadow: 0px 0px 20px 15px rgba(137, 42, 42, 0.526);
  padding: 40px;
  border-radius: 10px;
}

.AboutMe a {
  color: rgb(207, 207, 207);
}

.AboutMe img {
  border-radius: 10px;
  border: 2px solid;
  box-shadow: 0px 0px 50px 5px rgba(137, 42, 42, 0.526);
}
