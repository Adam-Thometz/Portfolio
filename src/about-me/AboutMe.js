import React from "react";
import TechStack from "../tech-stack/TechStack";
import techStackInfo from "../tech-stack/techStackInfo";

import "./AboutMe.css";

const CHROME_MUSIC_LAB_URL = "https://musiclab.chromeexperiments.com/";
const MUS_ED_LAB_URL = "https://musedlab.org/";
const TARA = "https://www.tarateachers.com/"
const SOUNDCLOUD_URL = "https://soundcloud.com/this-is-teslas-ghost";
const EIGHT_BILLION_MINDS = "https://8billionminds.com/";

const AboutMe = ({ style }) => {
  return (
    <article className="AboutMe" style={style}>
      <h2>About Me</h2>
      <p>
        I used to teach music at an elementary school for autistic children,
        which is where my love of coding comes from. Working in an underfunded
        school with few instruments, I had to think of creative solutions for
        bringing music into the classroom and technology was the most accessible
        and cost-efficient way of doing that.
      </p>
      <p>
        I became inspired to start coding while compiling resources for my
        classes, which included really fun apps like{" "}
        <a href={CHROME_MUSIC_LAB_URL}>Chrome Music Lab</a> and{" "}
        <a href={MUS_ED_LAB_URL}>MusEdLab</a>. I started with Python and
        eventually decided to enroll in a part-time bootcamp where I learned
        frontend and backend technologies. During my last year of teaching, I
        decided to make my own apps and ended up making Art Buddy, which is in
        continuous production and is used by some of my old colleagues!
      </p>
      <p>
        These days, I'm working on features for special education teachers at <a href={TARA}>TARA</a>. I'm also...
      </p>
      <ul>
        <li>building Art Buddy</li>
        <li>volunteering to help build <a href={EIGHT_BILLION_MINDS}>8 Billion Minds</a></li>
        <li>learning artificial intelligence</li>
      </ul>
      <p>
        In my free time, I <a href={SOUNDCLOUD_URL}>produce music</a>, cook,
        read, and game.
      </p>
      <h2>My Tech Stack</h2>
      <TechStack techStack={techStackInfo.map(tech => tech.name)} />
    </article>
  );
};

export default AboutMe;
