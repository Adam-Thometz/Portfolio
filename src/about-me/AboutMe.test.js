import React from "react";
import { render } from "@testing-library/react";
import AboutMe from "./AboutMe";

describe("AboutMe component", () => {
  it("renders without crashing", () => {
    render(<AboutMe />);
  });

  it("matches the snapshot", () => {
    const { asFragment } = render(<AboutMe />);
    expect(asFragment()).toMatchSnapshot();
  });
});
