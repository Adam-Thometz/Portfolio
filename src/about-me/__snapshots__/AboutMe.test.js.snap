// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutMe component matches the snapshot 1`] = `
<DocumentFragment>
  <article
    class="AboutMe"
  >
    <h2>
      About me
    </h2>
    <p>
      I used to teach music at an elementary school for autistic children, which is where my love of coding comes from. Working in an underfunded school with few instruments, I had to think of creative solutions for bringing music into the classroom and technology was the most accessible and cost-efficient way of doing that.
    </p>
    <p>
      I became inspired to start coding while compiling resources for my classes, which included really fun apps like 
      <a
        href="https://musiclab.chromeexperiments.com/"
      >
        Chrome Music Lab
      </a>
       and 
      <a
        href="https://musedlab.org/"
      >
        MusEdLab
      </a>
      . I started with Python and eventually decided to enroll in a part-time bootcamp where I learned frontend and backend technologies. During my last year of teaching, I decided to make my own apps and ended up making Art Buddy, which is in continuous production and is used by some of my old colleagues!
    </p>
    <p>
      These days, I'm looking for a role in frontend development. I'm also...
    </p>
    <ul>
      <li>
        building Art Buddy
      </li>
      <li>
        volunteering to help build 
        <a
          href="https://8billionminds.com/"
        >
          8 Billion Minds
        </a>
      </li>
      <li>
        building a website for a local psychotherapist
      </li>
      <li>
        learning in-depth computer science
      </li>
    </ul>
    <p>
      In my free time, I 
      <a
        href="https://soundcloud.com/this-is-psychic-hamster"
      >
        produce music
      </a>
      , cook, read, and game.
    </p>
    <h2>
      My Tech Stack
    </h2>
    <section
      class="TechStack"
    >
      <figure
        class="TechStackCard"
      >
        <img
          alt="Javascript"
          class="TechStackCard-img"
          src="javascript.png"
        />
        <figcaption
          class="TechStackCard-caption"
        >
          Javascript
        </figcaption>
      </figure>
      <figure
        class="TechStackCard"
      >
        <img
          alt="React"
          class="TechStackCard-img"
          src="reactJS.png"
        />
        <figcaption
          class="TechStackCard-caption"
        >
          React
        </figcaption>
      </figure>
      <figure
        class="TechStackCard"
      >
        <img
          alt="Redux"
          class="TechStackCard-img"
          src="redux.png"
        />
        <figcaption
          class="TechStackCard-caption"
        >
          Redux
        </figcaption>
      </figure>
      <figure
        class="TechStackCard"
      >
        <img
          alt="Python"
          class="TechStackCard-img"
          src="python.png"
        />
        <figcaption
          class="TechStackCard-caption"
        >
          Python
        </figcaption>
      </figure>
      <figure
        class="TechStackCard"
      >
        <img
          alt="Flask"
          class="TechStackCard-img"
          src="flask.png"
        />
        <figcaption
          class="TechStackCard-caption"
        >
          Flask
        </figcaption>
      </figure>
      <figure
        class="TechStackCard"
      >
        <img
          alt="PostgreSQL"
          class="TechStackCard-img"
          src="postgresql.png"
        />
        <figcaption
          class="TechStackCard-caption"
        >
          PostgreSQL
        </figcaption>
      </figure>
      <figure
        class="TechStackCard"
      >
        <img
          alt="Node"
          class="TechStackCard-img"
          src="nodeJS.png"
        />
        <figcaption
          class="TechStackCard-caption"
        >
          Node
        </figcaption>
      </figure>
      <figure
        class="TechStackCard"
      >
        <img
          alt="Express"
          class="TechStackCard-img"
          src="expressJS.png"
        />
        <figcaption
          class="TechStackCard-caption"
        >
          Express
        </figcaption>
      </figure>
      <figure
        class="TechStackCard"
      >
        <img
          alt="Jest"
          class="TechStackCard-img"
          src="jest.png"
        />
        <figcaption
          class="TechStackCard-caption"
        >
          Jest
        </figcaption>
      </figure>
      <figure
        class="TechStackCard"
      >
        <img
          alt="jQuery"
          class="TechStackCard-img"
          src="jquery.png"
        />
        <figcaption
          class="TechStackCard-caption"
        >
          jQuery
        </figcaption>
      </figure>
      <figure
        class="TechStackCard"
      >
        <img
          alt="Bootstrap"
          class="TechStackCard-img"
          src="bootstrap.png"
        />
        <figcaption
          class="TechStackCard-caption"
        >
          Bootstrap
        </figcaption>
      </figure>
      <figure
        class="TechStackCard"
      >
        <img
          alt="CSS"
          class="TechStackCard-img"
          src="css.png"
        />
        <figcaption
          class="TechStackCard-caption"
        >
          CSS
        </figcaption>
      </figure>
    </section>
  </article>
</DocumentFragment>
`;
